mesonbuild/compilers/compilers.py

```diff
@@ -349,6 +350,11 @@ def get_base_compile_args(options: 'KeyedOptionDictType', compiler: 'Compiler')
             args.extend(compiler.get_profile_use_args())
     except KeyError:
         pass
+    try:
+        if options[OptionKey('b_profile')].value:
+            args += compiler.get_gprof_args()
```

Comment:
Given you documented it as an option with multiple values, shouldn't it be passed to get_gprof_args()?

Reply:
Thanks @anarazel

I made the correction in the docs. It's intended to be Boolean.

Reply:
@anarazel It's not up to me. They asked me above to remove it.